requirements.txt

```diff
@@ -13,8 +13,7 @@ pyparsing==2.4.2
 python-dateutil==2.8.0
 PyWavelets==1.1.1
 redis==3.3.11
-redisai==1.0.1
-rmtest==0.7.0
-scikit-image==0.16.1
+redisai==1.0.2
```

Comment:
1.2.0?

Reply:
Updated to redisai 1.2.0. This means I removed the scikit-image, scipy, redis-py, ml2rt,and numpy dependencies, since those come from redisai. IMHO this project should really move to poetry.